src/simple-pg/sql/helper.js

```diff
@@ -0,0 +1,35 @@
+import sql, { raw, join } from 'sql-template-tag';
+
+export const _nowToInt = 'cast(extract(epoch from now()) as integer))';
```

Comment:
```suggestion
export const _nowToInt = sql`cast(extract(epoch from now()) as integer))`;
```

Reply:
Suggest renaming to `nowTimestamp`. IMO there's no need for the leading `_` here.